docs/scraping/overview.md

```diff
@@ -77,6 +77,12 @@ values are `v1`. *(Required)*
 - `azureMetadata.cloud` - The name of the Azure cloud to use. Options are `Global`
  (default), `China`, `UsGov` & `Germany`.
 
+### Azure Monitor Integration
```

Comment:
Can we move this to https://docs.promitor.io/v2.11/scraping/runtime-configuration/?h=azure+monitor#azure-monitor?

Reply:
Makes sense. I had to go back and make a code change.